drivers/generic/cpssEnabler/linuxNoKernelModule/drivers/ethDriver.c

```diff
@@ -3399,8 +3409,13 @@ int mvppnd_open(struct net_device *dev)
 
 	mvppnd_disable_tx_interrupts(ppdev);
 
+#if LINUX_VERSION_CODE < KERNEL_VERSION(5,19,0)
```

Comment:
Up until 6.1.x, netif_napi_add() wraps around 4:4 to netif_napi_add_weight(). so #if here should be < KERNEL_VERSION(6,1,0)

Reply:
Thanks i have incorporated this in revised patch.